bsconfig.json

```diff
@@ -27,7 +27,7 @@
   ],
   "suffix": ".bs.js",
   "namespace": false,
-  "bs-dependencies": ["reason-react", "bs-fetch"],
+  "bs-dependencies": ["reason-react", "bs-fetch", "wonka"],
```

Comment:
Make sure to build `wonka` so we can use it. @kitten do you recommend we take an explicit dep on `wonka` and hope it gets deduped, or just reference the one brought in by `urql`?

Reply:
Just take in the explicit dependency :+1: There's no peer dependency on it, but ultimately we only have to make sure that our version ranges match major versions.